docs/user_guide/alerts_dashboards.md

```diff
@@ -323,7 +323,7 @@ The Cyences app utilizes the data provided by the aforementioned vendor products
 
 The Vulnerability dashboard is designed to view the vulnerability count by severity, new vulnerabilities found over time, a vulnerability summary based on host(s), and a list of vulnerabilities. Splunk users can further use the drilldown capability to view a vulnerability list for a single host or a list of hosts affected by a vulnerability.
 
-Dashboard panels: 
```

Comment:
dashboard panels is more accurate actually.

Reply:
resolved in my PR.